modules/customer/acm_customer.install

```diff
@@ -7,6 +7,45 @@
 
 use Drupal\rest\Entity\RestResourceConfig;
 use Drupal\rest\RestResourceConfigInterface;
+use Drupal\user\Entity\Role;
+
+/**
+ * Implements hook_install().
+ */
+function acm_customer_install() {
+  // Add permissions into ACM role.
```

Comment:
I am not sure I would have put this into this module.
This is not really a role for customers but for a technical user.